cmd/ctr/commands/snapshots/snapshots.go

```diff
@@ -271,6 +281,115 @@ var removeCommand = cli.Command{
 	},
 }
 
+var createCommand = cli.Command{
+	Name:      "create",
+	Usage:     "create a new snapshot from an image",
+	ArgsUsage: "[flags] <image name>",
+	Description: `Creates a new snapshot from a specified image.
+
+When you're done, use the remove command.
+`,
+	Flags: append(commands.SnapshotterFlags,
+		cli.BoolFlag{
+			Name:  "print-mounts",
+			Usage: "Print the snapshotter layer mounts",
+		},
+		cli.BoolFlag{
+			Name:  "print-chainid",
+			Usage: "Print the snapshot's chain ID",
+		},
+		cli.StringFlag{
+			Name:  "platform",
+			Usage: "Create the snapshot for the specified platform",
+			Value: platforms.DefaultString(),
+		},
+		cli.StringFlag{
+			Name:  "key",
+			Usage: "Optional key to use for snapshot. Default is image name.",
```

Comment:
I think it might be okay to make this a required parameter just like the `prepare` command. Unless there is a specific use case where we don't care about the key with which the snapshot is created.

Reply:
Yeah that might be a good idea so that we can make sure people know what key to remove later

Reply:
Done :)